### PR TITLE
Added password hashing and compare logic in user schema

### PR DESCRIPTION
**Pull request by Pradip-001**
Added password hashing and compare logic in user schema

### CODE REVIEW

**Comment by Pradip-001** [comment]
please review @Anmolbansal1 , @ivantha

File: fact-bounty-server/routes/api/user.js [hunk 1]
```diff
@@ -28,18 +27,7 @@ exports.userRegister = function (req, res, next) {
...
-			bcrypt.hash(newUser.password, salt, (err, hash) => {
-				if (err) throw err
-				newUser.password = hash
-				newUser
-					.save()
-					.then(user => res.json(user))
-					.catch(err => console.log(err))
-			})
-		})
+		newUser.save().then(user => res.json(user)).catch(err => console.log(err));
```
Comment (Anmolbansal1) [inline fact-bounty-server/routes/api/user.js]: Remove semicolon here

File: fact-bounty-server/routes/api/user.js [hunk 2]
```diff
@@ -94,7 +81,8 @@ exports.userLogin = function (req, res, next) {
 					.status(400)
 					.json({ passwordincorrect: 'Password incorrect' })
 			}
-		})
+		});
```
Comment (Anmolbansal1) [inline fact-bounty-server/routes/api/user.js]: And here too

**Comment by Pradip-001** [comment]
@Anmolbansal1 , is force pushing a good idea for this minor change? or should i create new PR?

**Comment by Anmolbansal1** [comment]
Force pushing is better then creating a new pr everytime, now reopen this pr and force push your commit

**Comment by Pradip-001** [comment]
I am not able to reopen , the reopen button is disabled , seems like i have to create new PR again

**Pull request closed without merging**